imports/wasi_http/http_test.go

```diff
@@ -19,21 +19,23 @@ import (
 )
 
 type handler struct {
-	urls   []string
-	bodies []string
+	urls    []string
+	bodies  []string
+	methods []string
 }
 
 func (h *handler) reset() {
 	h.bodies = []string{}
 	h.urls = []string{}
+	h.methods = []string{}
```

Comment:
Mostly curious, are the empty slices semantically different than using `nil` here?

Reply:
I don't think so, I can use `nil` if you prefer. It's probably my own aversion to having null pointers in other languages where they are different.